item.py

```diff
@@ -1 +1,6 @@
-
```

Comment:
same comment here - just missing methods! could be something like examine or identify or change value - things like that!